src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Script.Extensibility;
+using Microsoft.Azure.WebJobs.ServiceBus;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    [CLSCompliant(false)]
+    public class ServiceBusScriptBindingProvider : ScriptBindingProvider
```

Comment:
This could be moved to the ServiceBus assembly, not sure yet. It really isn't a problem if this lives here, but it could be moved like the other extensions.